content/influxdb/clustered/admin/bypass-identity-provider.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Bypass your identity provider
+description: >
+  InfluxDB clustered generates a valid management token (known as the _admin token_)
+  that can be used in development and testing environments in lieu of configuring
+  and using an OAuth2 identity provider.
+menu:
+  influxdb_clustered:
+    parent: Administer InfluxDB Clustered
+weight: 207
+---
+
+{{< product-name >}} generates a valid management token (known as the _admin token_)
+for managing databases and database tokens and stores it as a secret in your InfluxDB namespace.
+You can use the admin token with the [`influxctl` CLI](/influxdb/clustered/reference/cli/influxctl/)
+in lieu of configuring and using an OAuth2 identity provider.
+
+{{% warn %}}
+#### Do not use in production
+
+This feature is for development and testing purposes only and should not be used
+in a production InfluxDB cluster.
+{{% /warn %}}
+
+## Configure influxctl to use the admin token
+
+{{% code-placeholders "INFLUXDB_NAMESPACE|DIRECTORY_PATH" %}}
+
+1.  If you haven't already, [download, install, or upgrade to `influxctl` v2.2.0 or newer](/influxdb/clustered/reference/cli/influxctl/#download-and-install-influxctl).
+2.  Use `kubectl` to retrieve the admin token from your cluster namespace's
+    secret store and copy it to a file:
+
+    ```sh
+    kubectl get secrets/admin-token \
+      --template={{.data.token}} \
+      --namespace INFLUXDB_NAMESPACE | base64 -d > token.json
+    ```
+
+3.  Update your `influxctl` connection profile with a new `[profile.auth.token]`
+    section.
+4.  In the `[profile.auth.token]` section, assign the `token_file` setting to the location of your saved admin token file:
+
+    ```toml
+    [[profile]]
+    # ...
+      [profile.auth.token]
+        token_file = "/DIRECTORY_PATH/token.json"
+    ```
+{{% /code-placeholders %}}
+
+In the examples above, replace the following:
+
+- {{% code-placeholder-key %}}`INFLUXDB_NAMESPACE`{{% /code-placeholder-key %}}:
+  The name of your InfluxDB namespace.
+- {{% code-placeholder-key %}}`DIRECTORY_PATH`{{% /code-placeholder-key %}}:
+  The directory path to your admin token file, `token.json`.
+
+## Revoke an admin token
+
+The admin token is a long-lived access token.
+The only way to revoke the token is to do the following:
+
+1. Delete the `rsa-keys` secret from your InfluxDB cluster's context and namespace.
+2. Rerun the `key-gen` job.
+3. Restart the `authz` service.
```

Comment:
@gunnaraasen or @hansenreid Are there specific commands we can give users to perform each of these steps? I assume the jobs can be run with `kubectl`, but I don't know the specific paths to the jobs.

Reply:
I can put together some commands here in a bit.

Reply:
Hey sorry to get back to this so late. Unfortunately I was running into some issues with rerunning the key-gen job, which might have to do with my kubectl version. _In theory_, the customer could run these commands, but I can't say with certainty on step 2 for rerunning the job.

1. Delete the `rsa-keys` secret from your InfluxDB cluster's context and namespace.
```sh
kubectl delete secrets/rsa-keys --namespace INFLUXDB_NAMESPACE
```

2. Rerun the `key-gen` job.
```sh
# List jobs to find the key-gen job pod
kubectl get jobs --namespace INFLUXDB_NAMESPACE
# Replace KEY_GEN_JOB with the key-gen job found with the previous command, and increment "key-gen-001" as needed
kubectl create job --from=job/<KEY_GEN_JOB> key-gen-001 --namespace INFLUXDB_NAMESPACE
```

3. Restart the `token-management` service.
```sh
kubectl delete pods -l app=token-management --namespace INFLUXDB_NAMESPACE
```